src/js/Properties.js

```diff
@@ -494,31 +494,16 @@ axs.properties.getTextFromHostLanguageAttributes = function(element,
                                                             existingComputedname,
                                                             recursive) {
     var computedName = existingComputedname;
-    if (axs.browserUtils.matchSelector(element, 'img')) {
-        if (element.hasAttribute('alt')) {
-            var altValue = {};
-            altValue.type = 'string';
-            altValue.valid = true;
-            altValue.text = element.getAttribute('alt');
-            if (computedName)
-                altValue.unused = true;
-            else
-                computedName = altValue.text;
-            textAlternatives['alt'] = altValue;
-        } else {
-            var altValue = {};
-            altValue.valid = false;
-            altValue.errorMessage = 'No alt value provided';
-            textAlternatives['alt'] = altValue;
-            var src = element.src;
-            if (typeof(src) == 'string') {
-                var parts = src.split('/');
-                var filename = parts.pop();
-                var filenameValue = { text: filename };
-                textAlternatives['filename'] = filenameValue;
-                computedName = filename;
-            }
-        }
+    if (axs.browserUtils.matchSelector(element, 'img') && element.hasAttribute('alt')) {
+        var altValue = {};
+        altValue.type = 'string';
+        altValue.valid = true;
+        altValue.text = element.getAttribute('alt');
+        if (computedName)
+            altValue.unused = true;
+        else
+            computedName = altValue.text;
+        textAlternatives['alt'] = altValue;
```

Comment:
Heh - that's an interesting question. Ideally we should use bracket notation for anything which is part of the implicit API, but since we're not doing that consistently, it doesn't really matter. Since this hasn't changed in this CL (just the indentation), I'll leave it for now, but we could consider revisiting the dot/bracket inconsistency at some point if we want to do some clean-up.